docs/Tutorials/Visualization.md

```diff
@@ -2,26 +2,74 @@
 Distributed under the MIT License.
 See LICENSE.txt for details.
 \endcond
-# Visualization {#tutorial_visualization}
+# Running and Visualizing {#tutorial_visualization}
 
-### 1D %Data Volume %Data
+### Creating an Executable from an Existing Source File
 
-Volume data from one dimensional simulations is visualized using the python
-script `Support/bin/Render1dData.py`. Run
-`python /path/to/Render1dData.py --help` to view the help text for the script.
-Let us consider a Burgers evolution where we want to visualize the solution,
-`U`, then in the run directory you can use
+SpECTRE source files for executables are located in
+`src/Evolution/Executables` and can be compiled by running the command
+`make EXECUTABLE_NAME` where `EXECUTABLE_NAME` is the name of the executable
+as defined in the `CMakeLists.txt` file located in the same directory as the
+source file. For example, to compile the executable that evolves a scalar wave
+using a three-dimensional domain, one runs the command:
+`make EvolveScalarWave3D`, which then results in an executable of the same name
+in the `bin` directory of the user's build directory. For more details on
```

Comment:
I don't understand the last sentence. Is it essential for building an executable? If not maybe it should be discussed instead in a tutorial creating a new executable

Reply:
It's not essential but it gives the user a concrete place to look where `EXECUTABLE_NAME` is used. I could replace this with a sentence redirecting the user to the creating a new executable tutorial, or remove the sentence entirely, do you have a preference?

Reply:
Oh, I don't think that pointing users to that file is what you want then. You should point users to the CMakeList.txt files and explain how to figure out the available executables. That file is the implementation of how executables are set up, which is not really informative unless you both know CMake and are having trouble using the functions defined in that file.